langchain/serpapi.py

```diff
@@ -75,45 +107,43 @@ def validate_environment(cls, values: Dict) -> Dict:
             )
         return values
 
+    async def arun(self, query: str) -> str:
+        """Use aiohttp to run query through SerpAPI and parse result."""
+
+        def construct_url_and_params() -> Tuple[str, Dict[str, str]]:
+            params = self.get_params(query)
+            params["source"] = "python"
+            if self.serpapi_api_key:
+                params["serp_api_key"] = self.serpapi_api_key
+            params["output"] = "json"
+            url = "https://serpapi.com/search"
+            return url, params
+
+        async with aiohttp.ClientSession() as session:
```

Comment:
This is not recommended, sessions should be reused if possible

Reply:
Also, the docs for this feature should probably mention this part of the openai python docs https://github.com/openai/openai-python#async-api

Reply:
will do -- thanks